src/dependency-manager/src/component-config/builder.ts

```diff
@@ -149,6 +149,12 @@ export class ComponentConfigBuilder {
       }
     }
 
+    if (config.debug && (typeof config.debug === 'string' || config.debug instanceof Array)) {
```

Comment:
Why do we need this new exceptional case suddenly?

Reply:
Removing support for some old shorthand where we let you set
debug: \<debug docker start command\>